.github/workflows/ci.yaml

```diff
@@ -0,0 +1,112 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
+
+        steps:
+            -
+                name: "Checkout code"
+                uses: "actions/checkout@v2"
+
+            -
+                name: "Install PHP"
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php-version }}"
+                    tools: "composer:v2"
+
+            -
+                name: "Validate composer.json"
+                run: "composer validate --strict --no-check-lock"
+
+            -
+                name: "Composer install"
+                uses: "ramsey/composer-install@v1"
+                with:
+                    composer-options: "--no-suggest --no-scripts"
+
+    tests:
+        needs:
+            - "composer-validate"
+
+        name: "PHP ${{ matrix.php-version }} + ${{ matrix.dependencies }} + Composer v${{ matrix.composer-version }}"
+
+        runs-on: ubuntu-16.04
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                dependencies:
+                    - 'lowest'
+                    - 'highest'
+                composer-version:
+                    - '1'
+                    - '2'
```

Comment:
can't we do as in travis as not make these multiply the matrix?
I having headaches with matrices multiplication, let's same some CO2 also.

Reply:
Sure, we will use composer v2 then, oK?

Reply:
OK!